content/build/hydrobuild.md

```diff
@@ -54,58 +54,103 @@ Docker Desktop 4.23.0 and later versions ship with a Hydrobuild-compatible
 Buildx binary. Alternatively, you can download and install the binary manually
 from [this repository](https://github.com/docker/buildx-desktop).
 
-## Connecting to Hydrobuild
+## Connect to Hydrobuild
 
-To start building with Hydrobuild, you must create a new builder using the
-`docker buildx create` command. The builder is connected to Hydrobuild through
-an endpoint that you specify. The endpoint represents a single, isolated
-builder. Builder endpoints have the following format:
+To start using Hydrobuild, you must first add the builder's endpoint using the
```

Comment:
Is there a word like "command" missing in this sentence? Or should the "the" removed? Might just be me being an English noob but it sounded a bit strange to me.

Reply:
Yeap, typo! Thanks for catching